tests/integration/test_permissions.py

```diff
@@ -157,6 +157,7 @@ def test_inspire_search_filter_public_collection(app, app_client, user_info, tot
     # Doing a client request creates a request context that allows the
     # assert to correctly use the logged in user.
     app_client.get('/search')
+
```

Comment:
These unrelated changes are why we get conflicts...

Reply:
my conflicts were due to `modules.workflows.tasks.arxiv` refactoring.